### ISSUE TITLE
[Feature]: Reorder Homepage

### ISSUE THREAD

**Issue by Pasithea0** [sussy-code/smov#21]
### What feature do you want to add?

<img width="930" alt="Screenshot 2024-05-05 at 8 58 16 PM" src="https://github.com/sussy-code/smov/assets/74743263/5a61e485-29f5-4be0-8607-1f457b5700fe">

I reordered the homepage: Continue watching is on top, and I removed the isBookmarked filter. 
Can PR if you want.

<img width="500" alt="Screenshot 2024-05-02 at 4 19 26 AM" src="https://github.com/sussy-code/smov/assets/74743263/f26d5c36-2cf6-4bdc-a9b1-e767cafaadd2">

Test it here:
https://smov-envious-git-main-pasithea0s-projects.vercel.app/

### Why do you want to have this feature?

Discord recomendation?

**Comment by Pasithea0** [comment]
actually, gonna have to do some bug fixes before a pr

**Comment by sussy-code** [comment]
But erm that makes no sense, bookmark takes priority because its bookmarked.

**Comment by Pasithea0** [comment]
Yeah, I don't know which is better, but I also think this makes sense too. It's how every streaming app works like Netflix, Prime, etc. Up to you

**Comment by Pasithea0** [comment]
Also you should have access to my vercel now, oops

**Comment by sussy-code** [comment]
nah i dont like it bookmarks is the priority over watching
